src/main/kotlin/org/jitsi/jibri/selenium/status_checks/LocalParticipantKickedStatusCheck.kt

```diff
@@ -0,0 +1,40 @@
+package org.jitsi.jibri.selenium.status_checks
+
+import org.jitsi.jibri.selenium.SeleniumEvent
+import org.jitsi.jibri.selenium.pageobjects.CallPage
+import org.jitsi.utils.logging2.Logger
+import org.jitsi.utils.logging2.createChildLogger
+import java.time.Clock
+import java.time.Duration
+
+class LocalParticipantKickedStatusCheck(
+    parentLogger: Logger,
+    private val clock: Clock = Clock.systemUTC()
+) : CallStatusCheck {
+    private val logger = createChildLogger(parentLogger)
+
+    init {
+        logger.info("Starting local participant kicked out call check")
+    }
+
+    // The timestamp at which we last saw the call transition from
+    // joined to kicked out
+    private val localParticipantKickedTime = StateTransitionTimeTracker(clock)
```

Comment:
I don't think you need/want `StateTransitionTimeTracker` here.  You don't want to wait for some amount of time after it was kicked to leave, right?  You just want to finish as soon as it was kicked?  I think you just want:
```
return if (callPage.isLocalParticipantKicked()) {
    logger.info(...)
    SeleniumEvent.LocalParticipantKicked
} else {
    null
}
```

Reply:
You are right, I will probably not need this. I applied your suggestion